File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/wishList/Success.java

package com.artmurka.artmurkaapp.data.model.pojo.itemlist.wishList;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.HashMap;

public class Success {

    @SerializedName("sort")
    @Expose
    private Sort sort;

    @SerializedName("goods_list")
    @Expose
    private HashMap<String, GoodsListDescription> goodsList;
    
    @SerializedName("goods_count")
    @Expose
    private Long goodsCount;
    @SerializedName("duplicate")
    @Expose
    private Long duplicate;
    @SerializedName("meta_data")
    @Expose
    private MetaData metaData;
    @SerializedName("msg")
    @Expose
    private String msg;

    @SerializedName("paginator")
    @Expose
    private Paginator paginator;

    public Sort getSort() {
        return sort;
    }

    public void setSort(Sort sort) {
        this.sort = sort;
    }

    public HashMap<String, GoodsListDescription>  getGoodsList() {
        return goodsList;
    }

    public void setGoodsList (HashMap <String, GoodsListDescription> goodsList) {
        this.goodsList = goodsList;
    }

    public Long getGoodsCount() {
        return goodsCount;
    }

    public void setGoodsCount(Long goodsCount) {
        this.goodsCount = goodsCount;
    }

    public Long getDuplicate() {
        return duplicate;
    }

    public void setDuplicate(Long duplicate) {
        this.duplicate = duplicate;
    }

    public MetaData getMetaData() {
        return metaData;
    }

    public void setMetaData(MetaData metaData) {
        this.metaData = metaData;
    }

    public String getMsg() {
        return msg;
    }

    public void setMsg(String msg) {
        this.msg = msg;
    }

    public Paginator getPaginator() {
        return paginator;
    }

    public void setPaginator(Paginator paginator) {
        this.paginator = paginator;
    }

}

File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/wishList/Success.kt
package com.artmurka.artmurkaapp.data.model.pojo.itemlist.wishList

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

import java.util.HashMap

class Success {

    @SerializedName("sort")
    @Expose
    var sort: Sort? = null

    @SerializedName("goods_list")
    @Expose
    var goodsList: HashMap<String, GoodsListDescription>? = null

    @SerializedName("goods_count")
    @Expose
    var goodsCount: Long? = null
    @SerializedName("duplicate")
    @Expose
    var duplicate: Long? = null
    @SerializedName("meta_data")
    @Expose
    var metaData: MetaData? = null
    @SerializedName("msg")
    @Expose
    var msg: String? = null

    @SerializedName("paginator")
    @Expose
    var paginator: Paginator? = null

}
